in/index.html.mako

```diff
@@ -185,6 +186,9 @@
           <th class="cost-ebs-optimized">
             <abbr title='Some instance types are charged additionally when configured for optimized EBS usage'>EBS Optimized surcharge</abbr>
           </th>
+          <th class="emr">
```

Comment:
`cost-emr` would be a more consistent classname for this column.

Reply:
Ack